drivers/gpio/gpio_brcmstb.c

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2024 Junho Lee <junho@tsnlab.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT brcm_brcmstb_gpio
+
+#include <zephyr/arch/common/sys_bitops.h>
+#include <zephyr/arch/cpu.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/gpio/gpio_utils.h>
+
+#define GIO_ODEN  0x00
+#define GIO_DATA  0x04
+#define GIO_IODIR 0x08
+#define GIO_EC    0x0c
+#define GIO_EI    0x10
+#define GIO_MASK  0x14
+#define GIO_LEVEL 0x18
+#define GIO_STAT  0x1c
+
+#define DEV_CFG(dev)  ((const struct gpio_brcmstb_config *)(dev)->config)
```

Comment:
These are no longer used.

Reply:
I'll remove unused GIO_* macros but it doesn't compile without DEV_CFG because it is used by DEVICE_MMIO_NAMED_MAP. Am I missing something?

Reply:
OK. I got it.

Reply:
@soburi These are internally used by DEVICE_MMIO_* macros. https://github.com/zephyrproject-rtos/zephyr/blob/cd2c7874b95eed7505620534b4b216ef42b7ce2b/include/zephyr/sys/device_mmio.h#L435